.github/dependabot.yml

```
@@ -0,0 +1,7 @@
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
```

As discussed on Slack, probably a less noisy tradeoff might be
```suggestion
      interval: "monthly"
```